test/integration/newapp_test.go

```diff
@@ -1,7 +1,11 @@
 // +build integration
-
 package integration
 
+/*
+
+// FIXME: This test is disabled because kubernetes switched to engine-api which
```

Comment:
it is not breaking us, just the test is broken... Cesar and I plumbed the
newapp code and docker code to work.

Sent from my phone

On 10 Jun 2016, at 10:32, Maciej Szulik <notifications@github.com> wrote:

In test/integration/newapp_test.go
<https://github.com/openshift/origin/pull/8856#discussion_r66622945>:

>  package integration
>
> +/*
> +
> +// FIXME: This test is disabled because kubernetes switched to engine-api which

If this is breaking us we should bump the issue to p0, imho.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/8856/files/52c53ae0596acbc9e5de573822c2db8286943913..d44c7099416d956d8a3b4d7cfe4e2071a648b547#r66622945>,
or mute the thread
<https://github.com/notifications/unsubscribe/AACsaFAT3-03_0O_RgMG_6fQYsAafWwbks5qKXVvgaJpZM4Ic8eO>
.